llap-tez/src/java/org/apache/hadoop/hive/llap/tezplugins/LlapTaskSchedulerService.java

```diff
@@ -3049,7 +3131,7 @@ boolean isUpdateInProgress() {
       return isPendingUpdate;
     }
 
-    TezTaskAttemptID getAttemptId() {
+    synchronized TezTaskAttemptID getAttemptId() {
```

Comment:
attemptId is final. Why would we need synchronization accessing it?

Reply:
Ack removed